native-stack/types.tsx

```diff
@@ -210,6 +210,13 @@ export type NativeStackNavigationOptions = {
    * @platform ios
    */
   gestureEnabled?: boolean;
+  /**
+   * Set this prop to true whenever you have translucent status bar in order for it to work correctly. Defaults to `false`.
+   * Only supported on Android.
+   *
+   * @platform android
+   */
+  translucentStatusBar?: boolean;
```

Comment:
`insetForTranslucentStatusBar` maybe? otherwise it feels like we are setting the status bar to translucent